beacon-chain/beacon-network-test-vectors.md

```diff
@@ -0,0 +1,38 @@
+  }
+}
+```
+## LightClientUpdatesByRange
+```json
+{
+  "6684738": {
```

Comment:
`LightClientUpdatesByRange` includes only one LC update for slot 6684738 right?

Reply:
No, it should be 4 updates, as is also shown in the content key. With the first one from the indicated slot.